core/domain/opportunity_services.py

```diff
@@ -55,6 +55,39 @@ def is_exploration_available_for_contribution(exp_id):
 
     return model is not None
 
+def get_translation_opportunity_summary_from_model(
```

Comment:
This name doesn't explain properly what this function does if I compare `get_translation_opportunity_summary_from_model` and `get_exploration_opportunity_summary_from_model` I would think that one returns translation and the other exploration opportunity summary, but they return the same thing with some alteration.

Reply:
Done. Updated the function name.